src/main/java/nl/bo/techiteasydeel1/services/TelevisionService.java

```diff
@@ -14,20 +19,28 @@
 @Service
 public class TelevisionService {
     private final TelevisionRepository televisionRepository;
+    private final RemoteControllerRepository remoteControllerRepository;
+    private final CIModulesRepository ciModulesRepository;
+    private final WallBracketRepository wallBracketRepository;
 
-    public TelevisionService(TelevisionRepository televisionRepository) {
+    public TelevisionService(TelevisionRepository televisionRepository, RemoteControllerRepository remoteControllerRepository, CIModulesRepository ciModulesRepository, WallBracketRepository wallBracketRepository) {
         this.televisionRepository = televisionRepository;
+        this.remoteControllerRepository = remoteControllerRepository;
+        this.ciModulesRepository = ciModulesRepository;
+        this.wallBracketRepository = wallBracketRepository;
     }
 
-    public TelevisionDto toTelevisionDto (Television television){
+    public TelevisionDto toTelevisionDto(Television television) {
```

Comment:
Je DTO heeft relaties, maar deze mapper doet niks met die relaties. Als je nu een TelevisionById op vraagt van een Teelvision die een relatie heeft met een WallBracket, dan krijg je "wallbracket:null" te zien in postman.